eng/pipelines/templates/stages/archetype-spec-gen-sdk.yml

```diff
@@ -195,15 +195,30 @@ extends:
                 $optional_params
             displayName: 'Generate SDK'
 
-          - task: Powershell@2
+          - task: PowerShell@2
+            displayName: 'Output variables'
             inputs:
-              filePath: $(SdkRepoDirectory)/eng/common/scripts/Save-Package-Properties.ps1
-              arguments: >
-                -ServiceDirectory $(GeneratedSDK.ServiceName)
-                -OutDirectory $(GeneratedSDK.StagedArtifactsFolder)/PackageInfo
-              pwsh: true
-              workingDirectory: $(SdkRepoDirectory)
-            displayName: Dump Package properties
+              targetType: 'inline'
+              script: |
+                $artifactName = "$(AddBreakingChangeLabelArtifactName)"
+                $artifactPath = "$(System.DefaultWorkingDirectory)/$(AddBreakingChangeLabelArtifactPath)"
+                Write-Host "Artifact Name: $artifactName"
+                Write-Host "Artifact Path: $artifactPath"
+                if (Test-Path $artifactPath) {
+                  Get-ChildItem $artifactPath -Recurse
+                }
+
+          - template: /eng/common/pipelines/templates/steps/publish-1es-artifact.yml
+            parameters:
+              ArtifactName: $(AddBreakingChangeLabelArtifactName)
```

Comment:
As we chatted lets only have one artifact that has a well-known name and not have the possibility of having multiple which can be confusing and require more downloading than needed.

Reply:
Updated to use well-known artifact name then read the content of it to determine the language and label action.